public/docs/_examples/architecture/ts/app/hero-list.component.html

```diff
@@ -2,8 +2,10 @@
 <h2>Hero List</h2>
 
 <p><i>Pick a hero from the list</i></p>
-<div *ngFor="let hero of heroes" (click)="selectHero(hero)">
-  {{hero.name}}
-</div>
+<ul>
```

Comment:
Why switch from `<div>` to `<ul><li>`?  That a distraction. I wanted _less code on the page_. I didn't want to have to deal with the bullets of a `<ul>`.

I'm not going to insist on reverting this.  But I think you are mistakenly trying to improve the code at the expense of the clarity of the Template Syntax. 

"LESS HTML IS MORE"